pkg/consensus/consensus.go

```diff
@@ -0,0 +1,32 @@
+package consensus
+
+import "github.com/CityOfZion/neo-go/pkg/util"
+
+const cacheMaxCapacity = 100
```

Comment:
is that smth declared in docs?

Reply:
No, it is a tunable parameter. C# implementation has a capacity of 100, so I decided to set it also as 100.

Reply:
I would say it's nice to have such comment there